src/applications/gi/containers/LandingPage.jsx

```diff
@@ -37,9 +43,19 @@ export class LandingPage extends React.Component {
     // Only search upon blur, keyUp, suggestion selection
     // if the search term is not empty.
     if (isVetTecSelected(this.props.filters)) {
+      this.setState({
+        searchError: false,
+      });
       this.search(value);
     } else if (value) {
+      this.setState({
+        searchError: false,
+      });
       this.search(value);
+    } else {
+      this.setState({
+        searchError: true,
+      });
     }
   };
```

Comment:
this.setState({
      searchError: !(isVetTecSelected(this.props.filters) || value),
    });

Reply:
Implemented.